### DISCUSSION TITLE
Using a trained GNN in the deployment phase

### DISCUSSION THREAD

**Discussion by AminKeshavarzi** [pyg-team/pytorch_geometric#9663] [Q&A]
Hi everyone, I am implementing a recommender system (user-item) using Graph Neural Networks (GNNs) (link prediction). I trained the network, and the accuracy (AURC) and (AUPR) for the validation and test datasets is good. However, when I want to use this trained model during deployment, I only have 6 features available for new users, whereas I had 18 features for users in the training, validation and test dataset. How can I handle this situation?

I am using PyTorch Geometric (PyG). In the deployment stage, how should I provide these 6 features to the model? Should I create a dataset similar to the test dataset (edge_index, edge_attr, edge_index_label, and edge_label)?
